pull-request.sh

```diff
@@ -43,6 +43,8 @@ check_events_json() {
 
 }
 
+export RETVAL=0
```

Comment:
Why do you define it up here as a global variable, but then again in specific scopes? Wouldn't it be better to define in specific scopes and avoid global variables?

Reply:
yes, this is leftover from an earlier version.